src/restaurant_inspections_api/services.clj

```diff
@@ -84,11 +84,11 @@
   (db/select-counties-summary))
 
 (defn- format-businesses-params
-  [params-map]
-  (assoc params-map
-         :zipCodes (when-let [zipCodes (:zipCodes params-map)]
-                     (str/split zipCodes #","))
-         :page (* (:page params-map) (:perPage params-map))))
+  [{:keys [zipCodes perPage page countyNumber]}]
```

Comment:
Ah I see what happened here. You previously had an `assoc` with an empty map `{}`. But the change doesn't do much. We passed from overwritting 2 keys to reconstructing a map... why?